Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -403,28 +368,33 @@ fun HomeCategoryTabIndicator(
     )
 }
 
-@OptIn(ExperimentalFoundationApi::class)
 @Composable
 fun FollowedPodcasts(
     items: PersistentList<PodcastWithExtraInfo>,
-    pagerState: PagerState,
     modifier: Modifier = Modifier,
     onPodcastUnfollowed: (String) -> Unit,
 ) {
-    HorizontalPager(
-        state = pagerState,
-        modifier = modifier
-    ) { page ->
-        val (podcast, lastEpisodeDate) = items[page]
-        FollowedPodcastCarouselItem(
-            podcastImageUrl = podcast.imageUrl,
-            podcastTitle = podcast.title,
-            onUnfollowedClick = { onPodcastUnfollowed(podcast.uri) },
-            lastEpisodeDateText = lastEpisodeDate?.let { lastUpdated(it) },
-            modifier = Modifier
-                .padding(4.dp)
-                .fillMaxSize()
+    val lastIndex = items.size - 1
+    LazyRow(
```

Comment:
Why was this changed? I assume the old one was to achieve snapping behavior, which should be possible without HorizontalPager if you need. But also there is a new Carousel in M3 which might be good also, I haven't actually looked into it though